samples/subsys/display/lvgl/src/main.c

```diff
@@ -86,7 +88,12 @@ void main(void)
 		hello_world_label = lv_label_create(lv_scr_act());
 	}
 
-	lv_label_set_text(hello_world_label, "Hello world!");
+	sprintf(version_str, "Hello world!\nLVGL version: %d.%d.%d\nZephyr version: %d.%d.%d",
```

Comment:
Another pimp "Hello world" example, I do not think it makes sense to inflate this. We also support controllers that are also installed in relatively small displays.

Reply:
I didn't think of very small displays. Should I simply cancel this PR?